controllers/scheduler_controller.go

```diff
@@ -234,11 +272,17 @@ func (r *SchedulerReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		}
 	}
 
-	// If the scale up is in state backoff, skip scale down. It is
+	// If any scale up is in state backoff, skip scale down. It is
 	// needed since in Backoff the cluster-autoscaler will not increase
 	// the cloudProviderTarget, e.g. : Backoff (ready=0 cloudProviderTarget=0).
-	if targetStatus.ScaleUp.Status == clusterautoscaler.ScaleUpBackoff {
-		down = 0
+	if down > 0 {
+		for i := range targetNodeGroups {
+			if targetNodeGroups[i].ScaleUp.Status == clusterautoscaler.ScaleUpBackoff {
+				log.V(1).Info("Skipping scaleDown due to scaleUp in backoff")
+				down = 0
+				break
+			}
+		}
```

Comment:
Does this mean that if only one of the ASGs is backoff, scaling downs are totally blocked?

Reply:
One of the targetted ASG, yes, and that's precisely what this PR aim to improve.
That was already the case before but with only one, now this PR changes that to make the scheduler
aware of a set of ASGs. 

If you want two ASGs to have contrary moves, e.g. if pods scheduled on them have nothing alike, 
you still can put them in different schedulers.